src/doom/p_enemy.c

```diff
@@ -1564,7 +1564,14 @@ A_PainShootSkull
 	P_DamageMobj (newmobj,actor,actor,10000);	
 	return;
     }
-		
+	
+    // [crispy] Lost Souls bleed Puffs
+    // needed because NOBLOOD flag in
+    // M_CrispyToggleColoredblood (m_crispy.c) won't apply
```

Comment:
I don't understand this comment. The code in `M_CrispyToggleColoredblood()` applies (or removes) the MF_NOBLOOD flag to all Lost Souls that are already on the map at the time of changing that config. All new Lost Souls will of course have to get that flag applied (or not) depending on the config value. However, this code here is not related to the one in `m_crispy.c`.

Reply:
Yes, but it doesn't work without the game restart atm (ie with current implementation if you got to let's say e3m2 with coloredblood turned off, and then turn it on without restarting, lost souls will still bleed blood).

Reply:
Could you still remove these comments, please? I understand where they come from, but they still feel a bit misplaced. After all, they only document my oversight, which becomes irrelevant once this is merged. Thank!